website/src/theme/SearchBar/index.js

```diff
@@ -0,0 +1,135 @@
+import React, { useRef, useCallback, useState, useEffect } from "react";
```

Comment:
Maybe we include an inline comment with a link to the source of this code? (I believe [this](https://code.davidawindham.com/david/til/commit/6e35b1bbe2804bb17bf91b32cc77f77303fcf7a0))

Reply:
...actually maybe I misunderstood - was this all just the content that was autogenned w/ swizzle? @chrisco255

Reply:
yeah it was autogenerated from Swizzle but I did add a little but of custom CSS